.github/workflows/bcny-build-nuget-package.yml

```diff
@@ -0,0 +1,142 @@
+name: "bcny-build-nuget-package"
+on:
+  workflow_call:
+    inputs:
+      arch:
+        description: "Name of the architecture amd64 or arm64"
+        required: true
+        type: string
+      platform:
```

Comment:
Any reason to not do both simultaneously as a matrix?

Reply:
Can you pass it as a matrix to the callable workflow/action? I'm not aware of this functionality and passing it as a JSON just to parse it on the other end seems like an overkill.